python/ray/tests/test_actor_failures.py

```diff
@@ -211,6 +212,66 @@ def get_pid(self):
         ray.get(actor.increase.remote())
 
 
+def test_named_actor_max_task_retries(ray_init_with_task_retry_delay):
```

Comment:
Maybe just parametrize the existing `max_task_retries` tests instead of writing a new test (and/or extend those tests)?

Reply:
Spent a few minutes trying this but it's pretty ugly because we need to do the calls from a remote actor.